ring_doorbell/const.py

```diff
@@ -40,6 +40,11 @@
 URL_DOORBELL_HISTORY = DOORBELLS_ENDPOINT + '/history'
 URL_RECORDING = '/clients_api/dings/{0}/recording'
 
+# chime test sound kinds
+KIND_DING = 'ding'
+KIND_MOTION = 'motion'
+CHIME_TEST_SOUND_KINDS = (KIND_DING, KIND_MOTION)
```

Comment:
Let's make a list instead a map

Reply:
I'm sorry, but can you please explain what the key and value of the map would be?

Reply:
```python
CHIME_TEST_SOUND_KINDS = [KIND_DING, KIND_MOTION]
```

Reply:
Why does it have to be a list? What i've used is a tuple which is basically an immutable list (not a map). Which is exactly what we need here.